pkgs/development/python-modules/colour-science/default.nix

```
@@ -0,0 +1,97 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  python3,
+  opencolorio,
+}:
+buildPythonPackage rec {
+  pname = "colour-science";
+  version = "0.4.6";
+  pyproject = true;
+  disabled = python3.pkgs.pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "colour-science";
+    repo = "colour";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-kjJc6D4jhvAJh6rIVvKO2bw++K3XlfjD4Djav6778lk=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
```

```suggestion
  nativeBuildInputs = [
```
These should all be listed in the package inputs.
Same below.